samples/python/kinetics/mechanism_reduction.py

```diff
@@ -25,42 +24,53 @@
 import cantera as ct
 import numpy as np
 import matplotlib.pyplot as plt
+from collections import namedtuple
```

Comment:
If the goal here is attribute-based access to results, I think a `dataclass` would be more idiomatic. I know we're not necessarily trying to showcase Python best practices, but for the sake of <10 lines, I'd rather avoid `namedtuple`s. Here's some context for this idea, although it's not totally specific to the case here: https://snarky.ca/dont-use-named-tuples-in-new-apis/